app/models/data.py

```diff
@@ -127,6 +124,21 @@ def pum(self):
     def fips(self):
         return fips_lookup(self.state)
 
+    @validates('totalTestResultsFieldDbColumn')
+    def validate_totalTestResultsFieldDbColumn(self, key, value):
+        """Validate the totalTestResultsFieldDbColumn value, used to calculate totalTestResults.
+
+        Acceptable values are either a valid CoreData column name or a known special keyword like 'posNeg'.
+        """
+        ttr_special_keywords = ['posNeg']
+        is_valid = value in ttr_special_keywords
```

Comment:
if I'm understanding this correctly this can be made a bit more concise:
`is_valid = value in ttr_special_keywords or value in [column.name for column in CoreData.__table__.columns]`

Reply:
Thank you I was trying to write that in a more Pythonic way and just made a mess of it